extension/src/frontend/view/EventDrawer.tsx

```diff
@@ -0,0 +1,170 @@
+}
+
+export function EventDrawer({ open, onOpenChange }: EventDrawerProps) {
+  const container = useContainerElement()
+
+  const events = useRecordedEvents()
+
+  const handleHighlight = (selector: string | null) => {
+    client.send({
+      type: 'highlight-element',
+      selector,
+    })
+  }
+
+  const handleNavigate = (url: string) => {
+    client.send({
+      type: 'navigate',
+      url,
+    })
+  }
+
+  return (
+    <RecordingContext recording>
+      <Dialog.Root modal={false} open={open} onOpenChange={onOpenChange}>
+        <Dialog.Portal container={container}>
+          <Dialog.Overlay />
+          <Dialog.Content
+            css={css`
+              position: fixed;
+              top: 0;
+              right: 0;
+              bottom: 0;
+              z-index: var(--studio-layer-1);
+
+              width: 35vw;
+              min-width: 300px;
+              max-width: 600px;
+              padding: var(--studio-spacing-4);
+              padding-left: var(--studio-spacing-5);
+              background-color: var(--studio-background);
+              box-shadow: var(--studio-shadow-1);
+
+              display: flex;
+              flex-direction: column;
+              gap: var(--studio-spacing-2);
+
+              &[data-state='open'] {
+                animation: ${slideIn} 0.3s cubic-bezier(0.22, 1, 0.36, 1);
+              }
+
+              &[data-state='closed'] {
+                animation: ${slideOut} 0.3s cubic-bezier(0.22, 1, 0.36, 1);
+              }
+
+              @media (prefers-reduced-motion: reduce) {
+                animation: none;
+              }
+            `}
+            onInteractOutside={(event) => {
+              event.preventDefault()
+            }}
+          >
+            <div
+              css={css`
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+                padding-left: var(--studio-spacing-2);
+              `}
+            >
+              <Dialog.Title
+                css={css`
+                  margin: 0;
+                `}
+              >
+                Events
+              </Dialog.Title>
+              <Dialog.Close
+                aria-label="Close event list"
+                css={css`
+                  background-color: transparent;
+                  color: var(--studio-foreground);
+                  border: none;
+                  padding: 0;
+                  cursor: pointer;
+                  padding: var(--studio-spacing-2);
+                  border-radius: 50%;
+
+                  display: flex;
+                  align-items: center;
+                  justify-content: center;
+
+                  &:hover {
+                    background-color: var(--studio-hover-color);
+                  }
+                `}
+              >
+                <Cross1Icon />
+              </Dialog.Close>
+            </div>
+            <BrowserEventList
```

Comment:
The list is not scrollable, so if it becomes long enough, there's no way to see the most recent events
<img width="1312" alt="image" src="https://github.com/user-attachments/assets/2019c477-b472-4ccf-8808-05233389053a" />